hsc/crawler.py

```diff
@@ -113,9 +115,18 @@ def login(self, username, password):
 		self.get_number_of_submissions()
 		return self.total_submissions != 0
 
+	def parse_script(self):
+		p = configargparse.ArgParser(default_config_files=['./user.yaml'])
+		p.add('-c', '--config', is_config_file=True, help='config file path')
+		p.add('-l', '--limit', help='limit to no. of solutions to be crawled')
+		p.add('-u', '--username', help='username')
+		p.add('-p', '--password', help='password')
+
```

Comment:
i think we should also add offset here

Reply:
As offset logic was still to be written
Would add offset in separate PR

Reply:
We can add offset in this PR only as it's not that big a change. @rishabhsingh971 @rajat19 Should I go ahead with this?

Reply:
I agree with @rajatgoyal715